WooCommerce/Classes/ViewRelated/ReusableViews/UnitInputTableViewCell.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+
+struct UnitInputViewModel {
+    let title: String
+    let unit: String
+    let value: String?
+    let inputFormatter: UnitInputFormatter
+    let onInputChange: ((_ input: String?) -> Void)?
+}
+
+/// Displays a title, an editable text field for user input and the unit of the text field value.
+///
+final class UnitInputTableViewCell: UITableViewCell {
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var inputAndUnitStackView: UIStackView!
+    @IBOutlet private weak var inputTextField: UITextField!
+    @IBOutlet private weak var unitLabel: UILabel!
+
+    private var inputFormatter: UnitInputFormatter?
+    private var onInputChange: ((_ input: String?) -> Void)?
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        configureTitleLabel()
+        configureInputAndUnitStackView()
+        configureInputTextField()
+        configureUnitLabel()
+        applyDefaultBackgroundStyle()
```

Comment:
Maybe turn off the cell selection style here? Because a different cell can be selected while the active cursor is in a different row, and the selection style isn't turned off in `didSelectCellForRow:`

Reply:
ah I haven't added the code to deselect the cell after `didSelect` a cell, will add in the next commit. I believe we'd want the default background style for the cells, will have Kylea's design review in the next PR and update the styles when needed